src/pages/research_profile/ResearchProfile.tsx

```diff
@@ -157,6 +159,20 @@ const ResearchProfile = () => {
             </Typography>
           </Box>
         )}
+        {(!!personBackground || personKeywords.length > 0) && (
+          <Box sx={{ mt: '1rem', display: 'flex', flexDirection: 'column', gap: '0.5rem', width: '70%' }}>
```

Comment:
noe av dette kan sikkert fjernes, på samme måte som du fjernet lignende på `ResearchProfilePage`?
```suggestion
          <>
```

Reply:
Beholdt `width`, men endret lengden til 80%. Litt usikker på hvordan jeg skal tolke skissene, da midtpartiet på forskerprofil skal bli litt mindre. 

![image](https://github.com/BIBSYSDEV/NVA-Frontend/assets/53341294/c0366916-6b61-40cc-8bd0-b8473829b25c)

Reply:
hvorfor brukes egentlig width her da? virker litt rart for meg at vi ikke ønsker at det skal gå helt ut. ser et argument med at vi ikke vil ha veldig lange linjer med tekst da, siden det er tungt å lese, men da burde vi ha en `maxWidth: "XXrem`, som er konstant uansett skjermstørrelse. Med prosent som nå blir det sikkert mye tom plass på mindre skjermer antar jeg?

Reply:
Ser poenget ditt, og har egentlig ikke noen sterke meninger om det. Om vi setter `maxWidth: "XXrem` blir det jo fort mye tom plass på brede skjermer igjen, men også lange linjer uansett. Syns forsåvidt det ser greit ut på mobil: 

![image](https://github.com/BIBSYSDEV/NVA-Frontend/assets/53341294/bb386c0e-4595-4e0f-b1c0-03172a42d614)

Reply:
vet ikke hva som er tenkt her jeg altså. eneste argumentet jeg ser er i hvert fall å unngå at selve tekstavsnitt blir veldig breie, og det gjelder desktop langt mer enn mobil. men der har vi mange svin på skogen fra før, så henger meg ikke særlig opp i det, så lenge det ser greit ut på både desktop og mobil